### ISSUE TITLE
HLS: Ignore unknown or unsupported system ID: 9A04F07998404286AB92E65BE0885F95

### ISSUE THREAD

**Issue by xusuona** [shaka-project/shaka-packager#1326]
**Packager Command**:

```
./packager-linux-x64 \
in=test.mp4,stream=video,output=test.video.mp4,drm_label=video,playlist_name='test.m3u8'  \
--enable_raw_key_encryption   \
--keys label=video:key_id=0B7H7DE341C74B3D88E35AFBD2BF0072:key=850RC88E356E35A00R7C962FE0052YU8   \
--protection_systems PlayReady  \
--hls_master_playlist_output master.m3u8 
```

**What is the expected result?**
Expect the Microsoft standard "ext-x-key" tag to appear in the generated test.m3u8, similar to

```
#EXTM3U
#EXT-X-VERSION:4
#EXT-X-KEY:METHOD=SAMPLE-AES-CTR,KEYFORMAT="com.microsoft.playready",KEYFORMATVERSIONS="1",URI="data:text/plain;charset=UTF-16;base64,xAEAAAEAAQC6ATwAVwBSAE0ASABFAEEARABFAFIAIAB4AG0AbABuAHMAPQAiAGgAdAB0AHAAOgAvAC8AcwBjAGgAZQBtAGEAcwAuAG0AaQBjAHIAbwBzAG8AZgB0AC4AYwBvAG0ALwBEAFIATQAvADIAMAAwADcALwAwADMALwBQAGwAYQB5AFIAZQBhAGQAeQBIAGUAYQBkAGUAcgAiACAAdgBlAHIAcwBpAG8AbgA9ACIANAAuADAALgAwAC4AMAAiAD4APABEAEEAVABBAD4APABQAFIATwBUAEUAQwBUAEkATgBGAE8APgA8AEsARQBZAEwARQBOAD4AMQA2ADwALwBLAEUAWQBMAEUATgA+ADwAQQBMAEcASQBEAD4AQQBFAFMAQwBUAFIAPAAvAEEATABHAEkARAA+ADwALwBQAFIATwBUAEUAQwBUAEkATgBGAE8APgA8AEsASQBEAD4AdgBHAFYAagBOAEsAZwBZAE0ARQBxAHAATwBMAGgAMQBWAGQAUgBUADAAQQA9AD0APAAvAEsASQBEAD4APAAvAEQAQQBUAEEAPgA8AC8AVwBSAE0ASABFAEEARABFAFIAPgA="
```

**What happens instead?**
However, the following error was reported during operation:

```
[0103/145509:WARNING:simple_hls_notifier.cc(495)] HLS: Ignore unknown or unsupported system ID: 9A04F07998404286AB92E65BE0885F95
```

I would like to ask if shaka can support it, or if there are other ways. Thanks!

**Comment by joeyparrish** [comment]
Seems reasonable.  Feel free to open a PR.

**Comment by joeyparrish** [comment]
Fixed in #1011
